src/main/java/com/iridium/iridiumskyblock/listeners/PlayerFishListener.java

```diff
@@ -24,7 +25,7 @@ public void monitorPlayerFish(PlayerFishEvent event) {
         User user = IridiumSkyblock.getInstance().getUserManager().getUser(event.getPlayer());
         Optional<Island> island = user.getIsland();
 
-        island.ifPresent(value -> IridiumSkyblock.getInstance().getMissionManager().handleMissionUpdates(value, "FISH", caughtEntity.getType().name(), 1));
+        island.ifPresent(value -> IridiumSkyblock.getInstance().getMissionManager().handleMissionUpdates(value, "FISH", ((Item) caughtEntity).getItemStack().getType().toString(), 1));
```

Comment:
Use Material#name(), it's just good practice 
```suggestion
        island.ifPresent(value -> IridiumSkyblock.getInstance().getMissionManager().handleMissionUpdates(value, "FISH", ((Item) caughtEntity).getItemStack().getType().name(), 1));
```